spec/classes/system_spec.rb

```diff
@@ -13,54 +15,55 @@
     }
   end
 
-  context 'default parameters', :compile do
```

Comment:
Why do you remove the `:compile` part?

Looks like this was removed in af8267945ab67ede82359d03252331a711640b63 (note the shared example is removed), but I'd argue we should rather reintroduce the shared example. This would give better coverage.